changelogs/fragments/2246-terraform.yaml

```
@@ -0,0 +1,4 @@
+breaking_changes:
```

Thanks to `get_version` function this is not a breaking change anymore :)
```suggestion
minor_changes:
```